docs/ad/index.md

```diff
@@ -52,7 +52,10 @@ In this case, a feature is the field in your index that you to check for anomali
 
 For example, if you choose `min()`, the detector focuses on finding anomalies based on the minimum values of your feature. If you choose `average()`, the detector finds anomalies based on the average values of your feature.
 
-You can add a maximum of five features for a detector.
+A multi-feature model correlates anomalies across all its features. It's difficult for the model to identify anomalies for an individual feature. Adding more features might negatively impact the [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) of a model.
```

Comment:
Can we make it less certain of the sentence: "It's difficult for .."  

In https://github.com/opendistro-for-elasticsearch/anomaly-detection/issues/397, I said anomalies for an individual feature would need to be stronger/bigger if we use a multi-feature model than if a single-feature model.

Reply:
Also, can we mention the curse of dimensionality (https://en.wikipedia.org/wiki/Curse_of_dimensionality) somewhere?

Reply:
Please review "A multi-feature model correlates anomalies across all its features. The [curse of dimensionality](https://en.wikipedia.org/wiki/Curse_of_dimensionality) makes it less likely for multi-feature models to identify smaller anomalies as compared to a single-feature model. Adding more features might negatively impact the [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) of a model."